Products/CMFPlone/profiles/default/actions.xml

```diff
@@ -494,6 +494,24 @@
       </property>
       <property name="visible">True</property>
     </object>
+    <object meta_type="CMF Action"
+            name="recyclebin"
+            i18n:domain="plone"
+    >
+      <property name="title"
+                i18n:translate=""
+      >Recycle Bin</property>
+      <property name="description"
+                i18n:translate=""
+      />
+      <property name="url_expr">string:${portal_url}/@@recyclebin</property>
+      <property name="icon_expr">string:plone-delete</property>
+      <property name="available_expr" />
```

Comment:
It should not be available if the recycle bin is disabled

Reply:
done in https://github.com/plone/Products.CMFPlone/pull/4168/commits/1ee283e2546e422b574fb8986eea92614fd3c808